audio/out/ao_audiotrack.c

```diff
@@ -815,6 +828,26 @@ static void start(struct ao *ao)
     mp_cond_signal(&p->wakeup);
 }
 
+static bool set_pause(struct ao *ao, bool paused)
+{
+    struct priv *p = ao->priv;
+    if (!p->audiotrack) {
+        MP_ERR(ao, "AudioTrack does not exist to pause!\n");
+        return false;
+    }
+
+    JNIEnv *env = MP_JNI_GET_ENV(ao);
+    if (paused) {
+        MP_JNI_CALL_VOID(p->audiotrack, AudioTrack.pause);
```

Comment:
thought I'd have to verify: since the feed thread and not synchronized could this cause audio to be paused before the intended point (cut off)?

Reply:
I'd expect `ao->driver->set_pause` to perform an immediate pause, just like what we do in the push-based AO.